### PR TITLE
remotecontrol: Move to context-aware robotjs fork for electron 14+

### PR DESCRIPTION
**Pull request by csett86**
Will be reverted as soon as https://github.com/octalmage/robotjs/pull/696
is merged upstream.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,4884 +1,8 @@
 {
   "name": "jitsi-meet-electron-utils",
   "version": "3.0.0",
-  "lockfileVersion": 2,
```
Comment (saghul) [inline package-lock.json:4]: Please undo this change, aka, run npm install with npm 8

**Comment by saghul** [comment]
LGTM, minue the downgrade in package-lock

**Comment by csett86** [comment]
Agree, the downgrade was due to the #130 not being merged… will redo package-lock update later.

File: package-lock.json (hunk 1 above)
Comment (csett86) [reply package-lock.json]: Done

**Comment by csett86** [comment]
Now with npm8, please review

**Review by saghul — APPROVED** [review]
LGTM, let's hope the failure was a fluke,  I retried the job.

**Comment by csett86** [comment]
I am also locally checking if I can reproduce it in a win10 VM... To me the build failure looks reproducible, and the strange thing is that with npm6 and node12 it was working, and on all other platforms its also working...

**Comment by csett86** [comment]
I finally found and fixed the error: prebuild has a dependency on node-gyp, which was then too old and failing with the non-descript erros. Updating prebuild (in the robotjs fork) helped.

**Comment by saghul** [comment]
Wonderful!

**Review by saghul — APPROVED** [review]
